docs/integrate_ultra.md

```diff
@@ -0,0 +1,40 @@
+# Integration with Ultra
+
+This section will guide you while you are integrating your application with Ultra. If you want to have a visual walk-through before following the detailed instructions, please head over to our [Ultra Demo](https://docs.google.com/document/d/1o3j1QRr4PofuJvWaFhsAN18vnLURFMEMBKpqD09Dt2Y/edit#heading=h.x9yfo0xokx3e) segment. You can open the Ultra playground application on your browser there and learn about implementation of the APIs.
```

Comment:
Links can point to another .md file, just put the name of the md file and it will be converted to a link